huaweicloud/services/dli/resource_huaweicloud_dli_queue.go

```diff
@@ -82,6 +83,33 @@ func ResourceDliQueue() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
+			// When the shared mode queue is offline, these parameter are mandatory for creating the exclusive mode
+			// queue and must be marked as mandatory in the document, but the ability to create a shared mode queue is
+			// retained (some regions may retain the ability to manage existing resources).
+			"elastic_resource_pool_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				Description: utils.SchemaDesc(
+					`The name of the elastic resource pool to which the queue belongs.`,
+					utils.SchemaDescInput{
+						Required: true,
+					},
+				),
+			},
+			"resource_mode": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IntInSlice([]int{resourceModeShared, resourceModeExclusive}),
```

Comment:
It is unnecessary to add function `ValidateFunc`.

Reply:
OFC